extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -710,11 +820,14 @@ In order to do so remove the existing `submitAdditionalPaymentDetailsResponse` c
 }
 ```
 
-## Step 7: Capture payment (required for Klarna)
+## Step 7: Capture payment (required for Klarna and Affirm)
 
 All Klarna payments [have to be manually captured](https://docs.adyen.com/payment-methods/klarna/web-component#capture) within 28 days after authorization, even if you have enabled automatic capture on your Adyen merchant account.
```

Comment:
I would not duplicate the docs, I think we could merge them into one sentence, right ?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/785/commits/27b3ec0a11d60a3642fc88f4c5acf4b64382daf3